packages/react-components/src/search/EventSearch/views/Table/index.js

```diff
@@ -142,7 +142,7 @@ const defaultTableConfig = {
       trKey: 'filters.measurementOrFactTypes.name',
       value: {
         key: 'measurementOrFactTypes',
-        formatter: (value, item) => <>{value.join(', ')}</>
+        formatter: (value, item) => <>{value || [].join(', ')}</>
```

Comment:
this probably should be

 formatter: (value, item) => <>{(value || []).join(', ')}</>

Reply:
Changed

Reply:
the problem still seems to be there

Reply:
Pushed

Reply:
the problem is still there :)

Reply:
I've fixed this myself so it can be merged